src/Tasks/Microsoft.Common.CurrentVersion.targets

```
@@ -4920,9 +4920,12 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     Leave the Clean cache file containing only the files produced in the current build.
     ============================================================
     -->
+  <PropertyGroup>
+    <IncrementalCleanDependsOn>_CleanGetCurrentAndPriorFileWrites</IncrementalCleanDependsOn>
+  </PropertyGroup>
   <Target
        Name="IncrementalClean"
-       DependsOnTargets="_CleanGetCurrentAndPriorFileWrites">
+       DependsOnTargets="$(IncrementalCleanDependsOn)">
```

Another option would be to make this

```suggestion
       DependsOnTargets="$(IncrementalCleanDependsOn);_CleanGetCurrentAndPriorFileWrites">
```

So that it's purely additive and we don't have to deal with the possibility of someone accidentally emptying/overwriting `$(IncrementalCleanDependsOn)`. I think I prefer that. Anybody want to talk me out of it?

---

Only downside I see is if they already have _CleanGetCurrentAndPriorFileWrites in the list or if they don't want it there for some reason. In any case: sounds good to me.